src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -121,7 +140,10 @@ public static ProcessInfo[] GetProcessInfos(string machineName)
                 string? processName = Interop.Kernel32.GetProcessName((uint)processId);
                 if (processName is not null)
                 {
-                    return NtProcessInfoHelper.GetProcessShortName(processName);
+                    ReadOnlySpan<char> newName = NtProcessInfoHelper.GetProcessShortName(processName);
+                    return newName.SequenceEqual(processName) ?
```

Comment:
case insensitive?

Reply:
No, that would be pure overhead

Reply:
Oops, misread. Does this suggest an API on `ReadOnlySpan<char>` like `string ToString(string unifywith)` or some better name..